python/ray/tune/tests/test_progress_reporter.py

```diff
@@ -246,21 +246,24 @@ def testProgressStr(self):
             t.trial_id = "%05d" % i
             t.local_dir = "/foo"
             t.location = "here"
-            t.config = {"a": i, "b": i * 2}
-            t.evaluated_params = t.config
+            t.config = {"a": i, "b": i * 2, "n": {"k": [i, 2 * i]}}
+            t.evaluated_params = ["a", "b", "n/k/0", "n/k/1"]
```

Comment:
Are evaluated params usually a list?

Reply:
No. For the purpose of the Reporter they are treated as an iterable (thus iterating through the keys), but you're right. Updated that in the last commit